### PR TITLE
feature(ui-ux): announcement Banner 2.0

### PR DESCRIPTION
**Pull request by siradji**
Signed-off-by: Suraj Auwal <siradjiawoual@gmail.com>

#### What kind of PR is this?:

/kind feature

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/components/Announcements.tsx [hunk 1]
```diff
@@ -194,6 +194,83 @@ export function AnnouncementBanner ({
...
 
+export function AnnouncementBannerV2 ({
+  hideAnnouncement,
+  announcement,
+  testID
+}: AnnouncementBannerProps): JSX.Element {
+  const { isLight } = useThemeContext()
+  const isOtherAnnouncement = announcement.type === undefined || announcement.type === 'OTHER_ANNOUNCEMENT'
+  return (
+    <ThemedView
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/components/Announcements.tsx:205]:
```suggestion
    <ThemedViewV2
```

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/components/Announcements.tsx [hunk 2]
```diff
@@ -194,6 +194,83 @@ export function AnnouncementBanner ({
...
+              dark={tailwind({ 'text-mono-dark-v2-900': isOtherAnnouncement })}
+              style={tailwind({
+                'text-orange-v2': announcement.type === 'OUTAGE',
+                'text-red-v2': announcement.type === 'EMERGENCY'
+              })}
+            />
+          </TouchableOpacity>
+        )}
+      {announcement.id !== undefined && (
+        <ThemedView
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/components/Announcements.tsx:245]:
```suggestion
        <ThemedViewV2
```

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/components/Announcements.tsx [hunk 3]
```diff
@@ -194,6 +194,83 @@ export function AnnouncementBanner ({
...
+  hideAnnouncement,
+  announcement,
+  testID
+}: AnnouncementBannerProps): JSX.Element {
+  const { isLight } = useThemeContext()
+  const isOtherAnnouncement = announcement.type === undefined || announcement.type === 'OTHER_ANNOUNCEMENT'
+  return (
+    <ThemedView
+      testID={testID}
+      style={tailwind('mt-9 relative mx-5 px-5 py-3 flex flex-row items-center border-0.5 rounded-xl-v2', {
```
Comment (pierregee) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/components/Announcements.tsx:207]: Can move `mt-9` and `mx-5` in Announcements.tsx by wrapping it in a View component? Composite screen also reuses this

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/components/Announcements.tsx [hunk 4]
```diff
@@ -194,6 +194,83 @@ export function AnnouncementBanner ({
...
+                'text-red-v2': announcement.type === 'EMERGENCY'
+              })}
+            />
+          </TouchableOpacity>
+        )}
+      {announcement.id !== undefined && (
+        <ThemedView
+          style={tailwind('absolute -top-2 -right-2 rounded-full')}
+        >
+          <ThemedIcon
```
Comment (pierregee) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/components/Announcements.tsx:248]: Clicking x icon doesn't have a press state
image

**Review by pierregee — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/components/Announcements.tsx (hunk 3 above)
Comment (kyleleow) [reply mobile-app/app/screens/AppNavigator/screens/Portfolio/components/Announcements.tsx:207]: let's revisit when we work on dex 2.0

**Review by pierregee — APPROVED** [review]

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**